lib/core/sync_job_runner.rb

```diff
@@ -52,14 +51,18 @@ def do_sync!
       begin
         Utility::Logger.debug("Successfully claimed job for connector #{@connector_settings.id}.")
 
-        @connector_instance.do_health_check!
+        # will be replaced when job claiming returns full job description object
```

Comment:
👍 I'll tackle this in https://github.com/elastic/enterprise-search-team/issues/3149

Reply:
👍